core/Run.ml

```diff
@@ -468,7 +470,7 @@ let ends_with_newline str =
   str <> "" && str.[String.length str - 1] = '\n'
 
 let print_status ~highlight_test ~always_show_unchecked_output
-    (((test : _ T.test), (status : T.status), sum) as test_with_status)=
+    (((test : T.test), (status : T.status), sum) as test_with_status)=
```

Comment:
Does ocamlformat allow no space before `=`?

Reply:
ocamlformat doesn't run on this repo. It's weird that I didn't see the missing space, I'm usually annoyed by these things.